web/src/features/fba/components/viz/FuelSummary.tsx

```diff
@@ -23,14 +23,23 @@ interface FuelSummaryProps {
   selectedFireZoneUnit: FireShape | undefined
 }
 
-// Column definitions for fire zone unit fuel summary table 
+const StyledHeader = styled('div')({
+  whiteSpace: 'normal',
+  wordWrap: 'break-word',
+  textAlign: 'center',
+  fontSize: '0.75rem',
+  fontWeight: '700'
+})
+
+// Column definitions for fire zone unit fuel summary table
 const columns: GridColDef[] = [
   {
     field: 'code',
     headerClassName: 'fuel-summary-header',
-    headerName: 'Fuel Type',
+    headerName: 'Primary Fuels',
```

Comment:
Is 'Primary Fuels' still the preferred header even though the table displays more than the top 3 fuel types by area?